eng/pipelines/common/templates/runtimes/run-test-job.yml

```diff
@@ -428,6 +428,16 @@ jobs:
           - jitstress_isas_x86_evex
           - jitstress_isas_x86_noavx512
           - jitstressregs0x2000
+        ${{ if in(parameters.testGroup, 'jitstress-isas-sve') }}:
+          scenarios:
+          - default_isas_arm64_sve
+          - jitstress_isas_1_arm64_sve
+          - jitstress_isas_2_arm64_sve
+          - jitstressregs_isas_1_arm64_sve
+          - jitstressregs_isas_2_arm64_sve
+          - jitstressregs_isas_3_arm64_sve
+          - jitstressregs_isas_4_arm64_sve
+          - jitstressregs_isas_8_arm64_sve
```

Comment:
This doesn't look to match the existing pattern, notice that `jitstress-isas-avx512` is separate from `jitstressregs-x86`

The naming pattern also seems off, I'd expect these to be `jitstressregs1_arm64_sve` and similar instead

Reply:
> This doesn't look to match the existing pattern, notice that jitstress-isas-avx512 is separate from jitstressregs-x86

i think the naming are not consistent anyway. I based this naming from `jitstress_isas_2_x86_noavx512`. Should that not be the case or am i missing something?

> The naming pattern also seems off, I'd expect these to be `jitstressregs1_arm64_sve` and similar instead

This I agree.

Reply:
> i think the naming are not consistent anyway.

For this I rather meant the grouping of stress jobs. We don't want to run too many by default if possible since they'll run in every PR

We rather want most stress jobs to run in the regular outerloop and the SVE AltJit job + maybe a core stress job to run in the regular CI while we're doing critical work